### DISCUSSION TITLE
More Keybinds, radius, teleport to map marker..?

### DISCUSSION THREAD

**Discussion by centaurxx** [Astropilot/ValheimTooler#10] [Ideas]
Hi!
First of all, thanks for this great tool and also for updating to the latest version of the game. I was thinking about a few additions that came to my mind to this well-made tool in order to make it even more versatile.

1.) Since there is a config file to customize the tools show/hide key I was wondering, if it is a possibility to also add a more keybinds to toggle a few of its "on/off-functions" within the tab: "Player -> General" (god mode, fly, no placement costs etc.).
This would be huge, considering it can be quite challenging to toggle in some situations.

2.) Another thing that came to mind was maybe some kind of radius setting for the ESP, since "show dropped items", "show pickables" and "show deposits" is quite unusable for me because of the stuttering. 

3.) Be able to teleport to map markers, not only players. This would be amazing.

One thing that doesn't seem to work as intended is: Misc -> ESP -> "show deposits". Some times they show up but most of the time they don't (huge copper deposits)

Thanks you so much!
Cheers

**Comment by Astropilot** [comment]
Hi @centaurxx !

Thank you for these interesting proposals!

1.) It's totally implementable! I hadn't thought about shortcuts but it's definitely useful.

2.) Yes I agree, currently the performance of ESP is quite poor, especially when many objects have to be shown. The principle of a radius seems to me a good compromise to have better performances. Concerning the ESP of the deposits, I confess that since the beginning it never worked properly, for lack of motivation I didn't work on it again but I'll see if I can correct it.

3.) TP on markers shouldn't be a problem to implement, I'll look into that :)

**Comment by Astropilot** [comment]
Hi @centaurxx ,

I just made a new release of the cheat (v1.7.0) with the following additions:
- ESP performance has been significantly improved! It still generates latency but it should be usable already. I added a radius system as well but it is not magic because the performance bottleneck is not due to the display unfortunately.
- The ESP for deposits has been fixed, it now works correctly (and does not display rocks anymore).
- It is now possible to teleport on the markers of the minimap!

Regarding keybinds, it will come in a next update.
